src/blog/_posts/2022-01-31-community-spotlight-january-2022.md

```diff
@@ -0,0 +1,43 @@
+---
+
+Welcome to our very first Cadence Community Spotlight update!
+
+This monthly update focuses on news from the wider Cadence community and is all about what you have been doing with Cadence. Do you have an interesting project that uses Cadence? We want to hear from you. Also if you have any news items, blogs, articles, videos or events where Cadence has been mentioned then that is good too. We want to showcase that our community is active and is doing exciting and interesting things.
+
+Please see below for a short round up of things that have happened recently in the community.
+
+## Community Related Office Hours
+
+On the 12th January 2022 we held our first Cadence Community Related Office Hours. This session was focused on discussing how we plan and organise things for the community. This includes things such as Code of Conduct, managing social media and making sure we regularly communicate project news and events.
+
+And you can see that this monthly update is the result of the feedback from that session! We are happy to get any feedback for comments you may have. Please remember that this update is for you  so getting your feedback will help us improve it. 
+
+We will be planning other Community Related Office Hour sessions so please watch out for updates.
+
+## Adopting a Cadence Community Code of Conduct
+
+Some of you may already know that our community has adopted this version of the [Contributor Covenant](https://github.com/uber/.github/blob/dcd96c52f2d1d839208315a2572cf37f48e52e96/CODE_OF_CONDUCT.md) as our Code of Conduct. We want our community to be an open, welcoming and supportive place where everyone can collaborate.
+
+## Recording from Cadence Meetup Available
+
+Please don't worry if you missed our online [November Cadence meetup](https://www.meetup.com/UberEvents/events/281975343/) because the recording is now available. You can find out more details about the meetup and get access to recordings [here](https://www.youtube.com/watch?v=pXgCd1BilLQ)
+
+## Cadence in the News!
+
+Below are a selection of Cadence related articles, blogs and whitepapers. Please take a look and fee free to share via your own social media channels.
+
+- [Meet Cadence: Workflow Engine for Taming Complex Processes](https://thenewstack.io/meet-cadence-workflow-engine-for-taming-complex-processes/)
+- [Spinning Your Workflows with Cadence](https://www.instaclustr.com/blog/spinning-your-workflows-with-cadence/)
+- [Instaclustr Joins the Engineering Team at Uber in Supporting Cadence, the Powerful Open Source Orchestration Engine](https://www.globenewswire.com/news-release/2021/12/07/2347314/0/en/Instaclustr-Joins-the-Engineering-Team-at-Uber-in-Supporting-Cadence-the-Powerful-Open-Source-Orchestration-Engine.html)
+
+## Upcoming Events 
+
+- [Webinar : What is Cadence? And is it right for you?](https://info.instaclustr.com/webinar-emea-what-is-cadence.html?_ga=2.191041518.510582234.1643223308-2138855655.1638190316)</li>
```

Comment:
the trailing </li> element should be removed.

Reply:
Thanks! I've removed it now.